console/src/modules/Console.rb

```diff
@@ -240,6 +241,26 @@ def Console
       nil
     end
 
+    # Console fonts map
+    #
+    # The map can be read from two different files:
+    #
+    # * `consolefonts_PRODUCT.ycp` where PRODUCT is the downcased product's
+    #   short name. For example, `consolefonts_sles.ycp`.
+    # * `consolefonts.ycp` as a fallback.
+    #
+    # @return [Hash] Console fonts map
+    def consolefonts
+      return @consolefonts if @consolefonts
+
+      # Read the file for this product
+      product_filename = "consolefonts_#{OSRelease.id}.ycp"
+      @consolefonts = WFM.Read(path(".local.yast2"), product_filename)
+
+      # Fallback
+      @consolefonts ||= WFM.Read(path(".local.yast2"), "consolefonts.ycp")
```

Comment:
Just to minimize the amount of `consolefonts.ycp` files, so `casp` and `sles` will share the same file. On the other hand, if some release miss the `id` field, we'll have a fallback.